docs/background/plot_bgcube_true_reco.py

```diff
@@ -0,0 +1,204 @@
+"""
+Script to produce plots comparing 2 background cube models.
+
+Details in stringdoc of the plot_bg_cube_model_comparison function.
+
+Inspired on the Gammapy examples/plot_bg_cube_model_comparison.py script.
+"""
+
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.units import Quantity
+from astropy.coordinates import Angle
+from astropy.table import Table
+from astropy.io import ascii
+from gammapy.background import CubeBackgroundModel
+from gammapy import datasets
+
+E_REF = Quantity(1., 'TeV') # reference energy
+NORM = 1
+INDEX = 1.5
+
+
+def power_law(energy, E_0, norm, index):
```

Comment:
Can you call [gammapy.spectrum.power_law_flux](https://gammapy.readthedocs.org/en/latest/api/gammapy.spectrum.power_law_flux.html) and [gammapy.spectrum.power_law_integral_flux](https://gammapy.readthedocs.org/en/latest/api/gammapy.spectrum.power_law_integral_flux.html) instead of re-implementing them here?